web/sites/default/config/webform.webform.report_row_obstruction.yml

```diff
@@ -195,14 +178,14 @@ elements: |-
       '#type': radios
       '#title': 'What type of signage?'
       '#options':
-        'Unauthorized No Parking signs or obstruction to prevent parking': 'Unauthorized No Parking signs or obstruction to prevent parking'
+        'Unauthorized No Parking signs or obstruction to prevent parking, including unauthorized curb painting': 'Unauthorized No Parking signs or obstruction to prevent parking, including unauthorized curb painting'
```

Comment:
On line 302, the conditional needs to be updated - currently form dead ends if you select this option